crates/repository/src/lib.rs

```diff
@@ -40,6 +40,16 @@ use spl_network_messages::PlayerNumber;
 
 const CONNECT_TIMEOUT: Duration = Duration::from_secs(5);
 
+#[cfg(not(target_os = "linux"))]
+const USE_DOCKER: bool = true;
```

Comment:
What do you think of using cfg! directly here?

Reply:
This PR now no longer depends on #1054, this is now removed.